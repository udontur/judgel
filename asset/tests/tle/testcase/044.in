4302 174
20258 8402 81632 13144 62280 74234 38056 27492 39144 50033 98296 36586 93852 17608 73126 86530 88922 25054 83059 28397 63565 21572 79565 84663 58985 25069 59759 46160 75732 12341 92074 56275 28888 58750 96456 46877 71365 94080 83018 23639 63461 81306 57770 41800 5854 745 28119 9199 53094 71267 37735 45888 98590 38033 75026 41125 12292 73372 59532 80111 15598 18276 77490 95190 64637 90063 6698 89957 24181 68779 82870 12896 55455 37699 70005 78091 83214 61888 80369 32323 84548 40984 67775 7246 42347 71304 58818 72246 96780 11273 93596 86777 31333 95609 17314 685 46016 33034 67517 55966 42100 45530 47817 75819 37957 19121 59579 86055 77423 60632 63723 72028 99871 61185 62563 58302 56945 98476 15883 35752 5928 73554 37878 95372 19807 12575 85031 77672 59641 59292 89279 89393 81757 87025 7369 93346 61554 19244 66174 24416 54171 45864 79644 60673 88400 39196 66285 45391 77903 52421 61862 50966 72811 83347 49783 69205 96208 44467 37009 55507 95111 70847 28930 75182 14367 16394 16463 93694 9818 57755 23394 79030 3954 95260 24475 25930 82077 93278 5917 73829 59001 92035 59884 56806 55210 85373 80690 93328 20591 45910 61828 60755 96807 80947 50174 36932 63860 27334 62669 52287 52800 37864 7707 74305 98442 42855 57354 92775 64655 55859 88628 46934 54465 32379 33964 22708 57723 49213 9346 29672 94494 88489 93475 38239 18726 88836 92166 68821 69347 60670 24203 25287 32387 30971 85486 53231 35438 44461 68435 63416 64858 32042 23046 6285 32029 67116 83461 26272 34569 30553 60213 34556 37952 80487 20309 71591 41845 78609 14441 946 24175 64590 44600 46679 59947 49552 69159 91036 4878 41702 45293 65742 55843 37284 91419 77710 88899 66285 60704 93896 94375 49825 86885 1692 13690 5393 59631 18047 78342 71908 81276 69991 76620 89925 23692 88345 15123 70414 46888 26688 44289 51842 21783 3478 21622 81847 53802 94419 67100 16129 54414 93519 73580 20351 2253 80143 48687 18691 95287 84467 67043 91086 18835 48009 89619 71553 81929 28443 53808 82196 40754 75344 56332 42591 7489 65332 60473 62556 65158 84246 66296 3867 41804 20087 77522 9428 71413 63096 22466 52762 70300 39925 11667 80108 37191 91275 12465 62790 64648 31406 10481 63237 48077 40458 42327 33896 94295 18812 42600 21120 35037 83979 94497 685 26015 89221 27629 43912 6195 20097 5233 47879 24624 24679 36288 2348 90400 30887 32077 33878 18695 98042 91736 86228 58140 46290 18324 3685 81894 1676 90739 45501 25912 71329 73428 40432 17680 32843 35280 16316 64392 83873 49848 72981 62513 89448 43970 31044 59753 11324 36746 48379 55829 39808 97937 86062 35254 66231 59363 34692 59873 44069 13895 93415 37963 70192 9246 69860 42544 6142 46354 67932 53553 93653 23060 71162 33724 90803 12862 70955 7697 97258 49988 69824 79755 81262 57826 12336 71772 44903 48552 98947 35579 5996 35866 54238 63549 98054 43446 15793 25578 79263 4694 28386 2565 14014 80365 43594 39754 31208 97638 87625 52836 62205 22936 12426 94702 95111 44893 73275 54215 74722 73880 94860 59273 15918 50687 3674 94459 53274 14297 95588 78716 20166 32543 11698 29344 80206 89817 12016 56619 72946 76262 61832 25376 33283 91042 24363 46508 47225 90945 95949 91344 68287 26215 98184 17493 60494 71776 36393 74154 84096 94206 49366 24738 86181 78582 96958 50046 84629 299 1426 81691 22080 85461 67813 29081 11953 7625 86874 94987 72988 54615 20205 56206 77822 54120 36768 6926 40779 32264 97876 40981 73767 14627 15728 57221 33559 87973 64529 86459 59160 92508 93677 83731 63584 9586 92994 60036 81524 11820 20515 95239 41471 92773 91783 97101 88872 76242 96812 58355 54998 49948 5121 27385 31325 95294 66823 53349 16021 41345 86297 58445 20444 26699 53283 32758 72913 16962 73730 90223 41249 67508 44883 3531 99395 13609 56606 68896 40817 74681 54393 77938 85084 2310 13815 85559 50243 28333 59502 10498 77077 58246 47226 83479 65758 8537 57546 93851 19614 69824 33331 91856 67624 18470 68531 20595 76155 45497 52373 46429 69433 75311 55111 22050 74248 59247 15216 90815 60793 8733 33207 67876 20528 35347 41116 21880 73417 31049 65404 48478 45772 45119 4402 15264 67418 82697 59557 68639 76671 42613 39215 98051 43332 69639 53996 29912 43599 14599 77529 93724 36739 16574 46205 10294 88439 65585 93207 87504 74967 78519 25319 5143 22091 65797 50616 38723 89926 65745 22199 96106 93461 28647 76762 32810 57370 64567 36111 80164 4860 70218 98662 37482 13672 21417 70841 20857 60823 50472 49461 79903 89234 73586 96653 21954 73995 27847 61018 63865 16751 68520 88489 20788 12780 25855 6665 13313 32425 55060 89589 66153 40882 24703 26364 22058 61545 64750 47879 48872 35834 47334 88878 1558 65674 42528 36137 8712 94644 52628 29173 37166 10458 29176 97715 82308 2795 86193 85510 44331 78916 13446 99312 22001 39763 31109 97263 77861 54627 29446 19536 91617 24265 14989 20428 44850 35392 16686 44277 44374 58054 77881 90571 84788 69184 47089 61163 90571 25540 92431 52634 37687 58795 12186 51600 88268 16806 73091 65150 69777 92816 20160 48081 74958 89633 95407 30251 95003 59908 63749 95396 64829 86846 7174 10801 15983 39555 9401 83288 5092 79697 55222 34206 54339 94544 21134 4366 3845 328 62593 72331 66875 45931 38304 79418 76532 59635 75411 18201 39939 69608 76868 90290 60969 5793 75031 77855 50446 47820 63225 70219 72425 1676 24053 50577 67902 88397 76620 26646 68776 95189 69764 8740 69547 24798 69640 25053 7683 36846 70738 70153 8131 22559 4442 86003 31323 27125 31714 93487 13684 85626 22623 81748 99838 89962 1187 39341 3433 4946 69284 67905 4285 33421 92395 44791 81747 81394 28883 63093 11700 82431 82096 16550 87539 25112 58172 26965 87228 49366 42863 10662 73275 87934 53995 40466 96553 43752 53253 86737 18922 20134 51669 39816 18586 57883 20228 21723 68173 6853 67451 47369 20811 52217 17087 15848 58058 83006 22042 10161 29620 25764 7695 22886 1867 8407 97647 29006 5244 74300 75793 75306 91536 91841 77537 6422 31110 18551 71709 51567 42115 52482 25282 3818 64653 54583 51658 50401 91135 71337 30113 45458 30445 11985 29794 24201 15575 38314 21400 80277 56938 57217 39027 53102 78605 30704 34944 15333 50150 63729 84575 29359 16179 99986 62920 92858 48537 40596 24289 59840 72071 64325 39467 20418 23882 83231 89803 78327 29906 57184 80638 85782 72263 91995 81260 23286 69071 50031 59687 60230 74169 1077 28255 19439 95549 50247 18552 54997 55170 27788 78159 5142 76503 35589 607 60908 47543 65596 32367 14823 59224 60968 20292 55161 93881 12329 72434 75271 79754 7024 31258 85827 66849 26910 28569 32141 57059 29084 49162 2774 13165 17504 50737 92837 30722 79849 28630 18349 49220 66099 78463 96219 27655 88899 53114 77433 62184 40528 32869 3834 7191 45411 70461 54026 39475 72300 89304 86259 65205 68843 42185 36809 28936 88002 79195 8986 41514 1988 84678 98700 82379 13089 7635 85062 62107 90588 98955 19453 36260 99872 61395 46215 24710 6314 89781 11817 13804 18141 73991 989 45787 99504 4867 93622 59833 91954 98652 96010 41559 98058 25374 29216 29264 59520 82295 75219 3865 33392 31296 42583 51956 13775 4587 86821 12102 91760 94210 25175 62815 87822 43779 39773 87127 10497 21869 3731 93796 31341 14664 39854 10506 62849 69199 86588 49995 18334 26576 24450 20467 14520 72169 7064 97756 34382 41225 63002 40954 92276 61 89619 35343 6282 50362 72481 76096 17774 74285 14233 77645 75484 81014 66599 14407 48392 64339 29205 72288 71749 90128 16705 32734 93924 80929 23691 65147 5365 45199 62223 95579 80855 28744 70424 24641 46186 80795 95608 48929 1626 92087 44523 59533 1594 94714 21564 83836 56268 47747 25603 94343 11263 33538 33344 42946 20706 20180 65087 71939 14581 88300 42402 56751 10472 68758 78089 55437 922 27042 82878 42238 77458 34353 80911 17812 90288 69196 90360 26739 14172 18097 22818 95993 58548 84653 33768 35964 73228 36449 49726 98286 45117 46941 69524 3416 19034 74515 1158 46294 52117 46267 90514 12157 86701 22453 4434 81332 30890 8219 57706 3731 31230 56552 75638 41507 10215 74925 56197 53478 29799 11175 91331 8978 96870 34767 54848 54214 99640 41179 78298 47751 53153 79407 70221 49521 38247 16414 75764 88633 52685 99679 27262 36064 18085 79155 87336 8461 26507 61311 79311 28182 54386 33296 70800 13111 62277 65002 6798 92970 71589 14048 23745 66277 50567 20570 19421 39639 49826 18931 57489 78067 25434 31876 56809 71082 64047 34106 35337 52403 10097 3158 62411 76245 13735 7447 36872 10182 15711 89351 5010 1761 61669 3401 17333 63859 86748 28552 34484 50261 45513 50898 84649 86336 41120 56514 16172 13108 76225 77660 2253 42199 12213 58907 23158 44493 55453 23978 50901 30780 28512 10370 18677 99703 29282 20908 32237 41008 9541 49362 94138 38029 24162 43990 54491 24328 17751 11797 83012 86222 55989 8706 23096 64749 99754 10555 43265 39585 92685 48939 71860 48057 14042 66476 47302 4513 90107 38235 24113 13466 9175 4263 94528 10778 31254 25798 79564 99726 2784 61639 63148 66584 69229 44098 90464 58107 74515 61468 6163 52685 1079 65700 90613 85529 35852 86728 30267 48841 33345 23113 7966 10752 41109 82083 10513 76701 94003 61159 784 68393 14116 43024 71345 70306 5973 18379 62116 2516 77232 16293 91797 50193 89268 86396 94317 79114 46907 84848 18071 89976 52731 46113 94302 6176 38721 7970 26413 54269 43377 50028 53012 21656 42141 94713 89406 74030 97026 56639 50836 91591 58743 32118 45223 13658 59215 35624 89530 99140 92764 77473 76549 44550 89532 91030 68932 61680 40397 54576 93182 51393 27948 41149 49868 42018 94413 50009 64091 59776 76995 74146 84944 12181 88356 72115 61277 55513 19302 37601 27501 48317 5100 56875 95666 50748 57515 93813 50169 37310 64829 39275 71450 82035 35593 38483 26734 76509 33368 15341 49221 29511 83834 99355 85785 52328 87515 29407 88433 53007 2557 4969 15546 89118 7557 45725 51294 94783 76599 9311 72529 29303 64862 32688 7617 71531 63747 65948 25199 61241 60692 7841 94461 21609 22637 47635 63787 34252 26037 87565 45966 98181 2715 54705 5420 18657 80316 50754 92749 90577 84209 67843 44665 90532 22079 63234 8096 96286 99466 49385 48945 72209 89422 30394 1676 12300 42482 85309 33798 27646 24634 59142 62576 43279 35345 60795 53633 13147 76450 28945 19111 84872 21983 17418 59879 68910 92701 73497 78712 98573 82025 10159 84076 95150 679 45540 34541 7703 94782 73796 13536 89721 52108 48022 81576 52966 43788 24117 88001 99710 82699 96571 97023 73539 22345 26366 40312 15706 59224 8250 2105 29499 20250 56681 99909 15381 86179 29838 20243 44278 58273 47226 71792 37939 25648 98064 50106 12227 39402 78263 50230 85271 93638 92019 80834 16446 27161 13888 95753 64695 69468 562 13898 87862 51322 93243 49472 31680 92645 96253 28002 82496 364 75466 74659 36425 82948 48754 28986 10647 2680 45127 54571 899 35173 67381 94944 42520 97740 75409 76853 39897 28436 8257 24640 38626 23442 51707 9904 91659 73211 13415 90391 63174 21188 35253 12433 38476 83737 24357 92889 40613 78650 57309 27854 22452 68097 22381 60140 41760 49571 43039 45265 63156 97963 12698 51153 10350 36710 57167 75383 99619 30565 45750 1024 19699 26454 6174 4143 69562 42103 45989 45875 58269 77335 59581 9078 93477 9190 41621 34028 2236 96784 60383 85442 9469 28691 57343 79681 94327 6876 5970 50826 45747 86343 63294 61251 31988 84912 45653 86902 89778 85190 16363 12271 63121 28959 99717 65100 16328 51067 90077 2472 21309 81032 73901 75741 73114 30379 13981 11605 75396 35052 2247 13455 27247 46445 3441 87122 42455 57428 65704 22289 14437 5504 65314 13807 44963 26655 28574 42864 80070 79500 67703 82334 68083 86985 8351 5993 96072 70547 22499 91063 87746 61665 99897 7081 15079 14360 24238 36791 76515 63872 78411 64550 13340 57916 95445 60090 54241 17531 30089 45641 32357 79915 38390 95551 79159 41318 22851 7683 98921 22246 23632 9363 51381 60327 30395 56429 79988 11065 47840 21857 92078 61269 52003 90991 10296 26615 72737 90656 27156 90063 46792 19695 36356 49441 8436 50449 65547 56423 9360 207 89300 57866 38116 35964 73060 36300 71263 29747 73859 58312 13955 32533 57982 7737 63030 31901 18466 51577 66628 62555 29474 34109 43934 26602 36243 32043 20862 45640 81412 9471 19750 55781 17132 84921 88264 1998 32441 77953 58124 36260 83106 45128 69707 9393 16270 37358 75977 83368 62232 92293 42393 91870 52767 38560 8432 91319 65003 95656 59828 95372 16233 94716 83492 82432 23262 36048 34809 40627 50456 85858 71349 47457 73938 59480 807 86013 66809 67523 51928 90432 54872 799 80084 88432 30773 97105 20282 76862 73242 54572 58351 7248 46585 38534 38816 76341 28131 58105 52077 36229 60834 39673 35965 96334 72965 20546 10632 3107 63415 31071 3945 84092 29451 42102 12554 89928 18258 99405 54136 12602 95190 32878 63205 96845 59007 94553 20876 35152 80037 66012 88167 52692 18859 18672 94505 92599 95852 62993 34556 63144 83668 36519 7932 31759 19472 82182 82004 92070 84564 41476 42492 33698 16412 63268 94002 46568 23550 77813 87689 51236 61247 32521 28307 24118 71139 4955 87959 58068 85093 95753 62862 32532 40227 52677 23402 18708 38409 73418 96454 98920 73916 93740 42195 94859 74133 63550 46677 89299 85310 58430 70104 47383 11360 64713 2185 9505 81882 68303 96915 22222 82260 7987 89567 58231 53873 77539 89938 21088 79290 60704 99249 85373 95645 99081 56605 13796 93039 68757 80793 44291 95510 72402 57034 6711 72180 66690 11921 24102 90174 56176 94841 92885 9663 30559 60195 50119 67894 98157 86825 18176 1000 31971 42450 45266 97545 92687 52939 17080 88955 52564 44250 45168 48531 80798 52891 80020 50271 90761 44509 35250 26977 3768 6731 89254 68864 60334 33721 99591 16917 49376 83631 15642 68026 41509 43753 96500 73405 53792 22668 99427 91212 56439 15622 79736 67044 5862 63527 86593 41813 48317 10444 68191 66990 86052 8003 42951 64290 8093 46679 35958 5349 47148 59470 60981 11318 21561 98679 84320 23782 76430 28162 63939 22999 72883 22190 12692 22483 1708 89178 45031 73741 45093 94400 33028 42628 72833 45672 71118 10133 4775 54950 98900 1111 56928 79862 36116 24985 58654 51245 28857 93822 17225 94748 31709 45124 32749 5931 16413 51799 58911 78747 6487 51300 47777 96641 83713 5103 80936 36962 37346 6555 63312 71411 50306 99854 85108 65506 36733 57320 85398 59022 42433 35641 96666 58047 8000 88024 66863 39005 74928 30126 24604 1644 43260 34674 50949 89711 83257 64235 45426 72322 66444 29183 75886 83815 75494 23340 45843 67052 54174 98150 31328 71446 22905 64433 5266 32494 76012 30484 14829 82464 94430 5658 65067 77637 31425 80287 52233 69214 94255 13880 57563 92662 47973 79255 38625 88858 55496 86635 96178 25970 66832 28836 38861 36208 32334 37396 63596 64504 31815 4093 60245 74507 94898 29752 11421 60909 31208 5110 30430 65992 59396 68497 9047 73327 72541 69262 14990 76605 41676 1315 88615 98895 27230 39010 45196 67929 63021 65239 98971 60927 46096 55184 26107 52881 28504 84258 26376 92532 70970 78454 74844 15099 31442 34211 396 58464 32600 76413 52362 29459 87624 86839 79200 79047 55395 17810 3211 83197 90599 35096 37368 72766 84215 73574 50096 42132 95299 44823 61275 49921 66219 24087 96718 10573 79747 5641 45730 56120 63376 74720 33032 9906 86350 51835 93819 37654 14544 67204 78590 1427 69568 25568 86497 12841 33440 26083 13610 35323 73027 29749 97244 41302 57531 86754 20122 13128 69549 48554 86471 5561 41841 842 16621 41741 66017 26645 75003 18255 56488 98046 59579 29023 686 36070 91624 10204 254 98361 51586 87951 52711 57158 45190 99479 26030 96944 8051 30155 88431 14805 89966 39990 30026 59888 83128 11071 22660 85247 68038 43572 25051 14044 73475 21468 72207 805 95046 30776 95361 40340 59143 94099 76370 38003 24593 25791 4026 29616 96363 62648 77848 9841 76872 69978 32884 53336 99360 8086 30549 34190 56986 50010 3692 36215 2463 14046 64352 52663 65158 54925 17675 62782 99302 83736 63818 3210 74781 65856 15914 97408 50791 3746 33268 57328 93075 14023 69534 97972 7830 77159 79786 18905 50825 68332 70643 28513 71933 5389 31548 28678 26881 32145 94603 13368 24173 20485 87740 93646 79093 39749 86860 29094 90657 31133 80518 80508 25262 38077 17834 15166 90936 49588 86873 9400 40972 4407 97402 67961 69627 45420 58678 42617 82997 95518 84325 61922 23645 23017 47465 72470 13058 83472 53544 81490 25132 76197 39587 78772 85872 35595 29933 31629 72247 25753 26983 65568 49609 29997 79747 55589 50731 19101 82095 38478 11666 61460 36755 9781 70924 94283 234 58224 21744 13676 19915 54272 54494 49918 44961 12334 69520 62849 33420 81194 67163 71087 71950 83211 52714 94187 86208 51199 59963 30793 10239 32498 46717 43737 12278 88147 22231 54816 94594 37265 9467 25370 87218 44022 85746 32986 91206 45424 50838 84294 79508 8836 92443 82734 77854 57061 65709 10983 54773 33746 214 98785 17860 1338 26642 79282 21895 76915 45797 69042 41551 39589 28531 66585 32814 80730 25388 69469 64346 91684 17937 23774 24004 1251 58524 37988 57654 27836 43637 41030 99132 11024 10599 45847 42117 73451 95577 60199 37836 96767 421 81827 76624 37477 48187 32512 44248 4937 33768 14910 12003 72168 92252 29479 86780 19828 531 807 70579 2150 6809 59500 88800 48680 50645 58457 13285 31256 24179 14673 7105 35188 35187 7059 11179 36476 6570 68559 7185 9947 11457 15349 23489 67005 89438 3379 5385 77811 8587 61126 11533 94416 88117 79349 63362 17438 67143 65479 13570 48796 43214 15075 72230 77048 61413 82856 98187 48130 12138 78796 77321 49199 45985 54824 52525 30218 49194 69653 99073 86110 11907 29406 4560 38905 27098 48234 16563 29829 71415 19828 47229 33037 56584 35292 80736 40984 3660 11887 43682 2978 17874 59561 77594 46393 37961 40401 46323 76867 3859 89496 12793 64718 93570 12405 26067 66681 60909 33474 67598 72183 17033 24687 84499 50106 55245 38753 1371 27823 97522 94750 3270 12402 82305 42829 10246 75435 2045 294 92964 93117 30876 80348 66074 75806 54752 48629 81089 19986 31410 21441 8127 24188 81676 9276 33967 33382 19059 66686 77217 26004 80759 83808 35484 75723 53101 99527 44802 79076 51127 4107 10843 66444 1757 58879 99936 11543 33620 93485 38965 83816 28782 70601 5385 79706 78619 8973 82309 6453 26165 3416 15534 99462 41585 2634 83679 70632 33495 34660 64381 91345 14314 91845 604 7487 44238 11742 46153 91663 78708 49638 98774 46805 77810 36151 8374 4886 37029 99783 97121 18344 53169 90776 43775 79962 55291 91282 58265 36735 33677 18574 81188 68007 51685 45490 16465 35876 47137 61435 84639 3924 70297 13754 45738 76632 73484 44113 17067 28892 59360 56379 94211 5104 90666 55775 97467 65337 11201 52263 12743 99289 46989 44254 82511 99200 91127 62333 84863 18199 51479 77859 14184 27238 47455 21424 87735 95774 68742 3495 60659 65825 29233 73270 86641 61105 55236 52489 9785 88303 14699 94316 5175 76340 63690 88854 90553 70252 37116 43995 58206 50645 209 46873 67665 71858 27822 32769 97699 94973 78184 50311 90018 67244 29613 97365 50380 9085 79760 34476 75969 15649 44235 78538 13620 60993 92684 70220 87418 34424 37835 66659 14232 60540 40638 53776 5055 24126 35212 24690 45043 54849 55849 42038 70033 9691 83515 77308 53660 38898 30842 26318 30724 71985 85046 58668 42563 78949 6406 67968 99899 24210 72655 92341 14466 98889 75856 4721 68815 6241 66957 99402 27506 2198 25963 91184 93036 33802 62716 7286 86454 88560 24082 49389 15477 3227 31023 74562 86564 78421 39289 32126 81037 67441 45414 49781 76186 38556 48261 35744 87869 17755 70730 5973 73479 29525 22509 26364 67197 99765 7355 59774 35548 28233 60209 81268 97471 79902 78709 24141 67396 1515 52271 18724 95731 12562 48129 50601 28011 34427 26278 20520 2862 54910 34864 32635 9533 18469 35440 27889 37779 67468 83657 73873 54589 58174 74230 24590 82957 40336 30723 52577 67480 67306 24906 98766 21670 3154 5358 32914 48180 63857 74696 34176 70755 33931 64034 96675 85073 60491 56333 91726 3891 26231 86281 49149 878 3695 1830 78526 78322 21058 97024 71620 655 6180 99668 13522 28994 21932 59385 95442 35620 27252 38814 5917 36418 51618 75188 47896 34414 66604 63527 15511 85363 55788 32821 53373 95363 57138 9017 31357 5198 64336 64535 17273 42576 57144 64638 75668 65389 95358 13243 30457 4695 98145 92116 70799 20120 40591 42784 44951 1553 24068 41363 92914 31473 51391 9199 12343 26336 72895 80802 3473 89285 4846 12288 72471 55904 27292 15264 38761 16856 89747 46216 76916 1528 52090 62066 92984 31666 90053 7727 29730 71559 63954 62323 14930 10807 50223 44304 60788 24403 17767 89451 59609 36579 419 4794 27531 43665 89604 37158 3055 85862 99800 72923 83044 32418 61366 6467 37267 27694 50664 39467 70337 58004 76356 61703 50791 2172 47421 28917 24237 59473 75770 47759 43745 76222 79147 92516 61931 61208 67992 13602 86297 1445 55169 98970 67450 5186 23911 90041 38959 75692 44896 5193 65917 37980 39800 24912 89693 64829 85645 11707 89907 72005 85669 50165 12147 8457 85178 52316 7957 36805 36670 81743 45112 28338 32662 64319 97557 962 58583 47893 35336 17566 61180 73078 19818 65965 42310 5828 18514 74840 46129 3862 14275 28987 65320 66811 53099 99660 63518 31371 70684 99655 76122 55076 13637 81758 40062 20858 57999 72478 69606 58106 80344 97482 33195 48808 13422 30008 83184 61461 8519 81227 38821 9460 59680 18588 48302 99966 56980 21331 1873 82732 35805 2198 98866 18265 59025 23272 33215 55206 67283 22580 3494 73383 7703 83885 56832 76327 87092 2334 48254 83842 70605 68314 1914 95259 35 39070 91930 87196 36178 42190 58231 60240 90551 86680 9693 21595 62772 89059 45570 5502 25612 20111 6812 81452 12939 61724 9538 25778 12110 25209 6220 66780 19309 73306 18349 41056 84736 12297 85658 64831 28741 88428 40129 27445 68355 88751 95346 45979 73501 11111 57053 99825 16462 72387 87384 56960 78640 46793 31558 82423 98786 74039 76036 70315 28712 37979 16297 25629 27196 85908 98571 99589 37963 48699 93320 65074 4839 16867 55139 33127 56537 10440 65397 24542 33642 76752 60743 2962 74492 81599 34886 46735 79764 3898 1689 22690 93628 75385 63222 49857 13411 21709 17210 24025 72014 74288 36842 50376 22421 11314 87625 83577 51319 14691 81704 4545 23187 51891 70091 71880 87275 2870 40897 81800 88893 76126 67590 90450 1872 64853 71266 54043 12707 46852 57181 99631 89104 46608 61566 70759 34976 64832 26676 51788 40410 93356 97069 36042 52077 54705 24250 60087 41913 50928 55269 84845 54656 57597 31526 38770 9816 62140 5768 12138 85280 65272 12567 18588 94436 16153 86667 61613 43448 76512 27042 35642 82722 36216 8078 93146 65871 89204 73661 18426 22783 98757 67424 87423 80383 60982 93881 41602 6459 81045 31380 49391 5458 64063 11791 39316 53686 44429 99138 33016 76026 99345 78080 32182 90510 92997 82113 51375 42120 998 38994 69039 5419 87420 67486 74720 44485 22840 553 42780 77034 19232 12798 6057 53137 75734 37497 32810 56034 10991 5819 30777 27163 15453 63849 57852 5969 45223 89625 13472 95798 83392 64286 75723 84733 77812 97906 8701 78789 36894 95916 73034 39968 81730 93709 6826 25590 91158 59700 7998 80723 74785 20300 83901 64959 84828 19543 69029 77261 31187 68705 66121 70609 81152 56948 39640 28365 94096 29817 4801 4753 59795 36127 69528 38133 31217 24963 73238 64040 96486 24290 60739 2161 90387 19319 49009 67110 70316 61706 3055 73593 2368 56087 74569 18952 9502 17809 41557 72996 703 35610 71120 42820 16188 98490 75931 88413 63577 34978 64071 6534 24225 4775 51115 28367 86757 10902 47983 34170 14517 19169 57372 97863 27898 81633 87657 15323 86132 25638 59096 68896 63598 39391 33247 26645 10242 28318 5379 70716 82198 73678 33782 72312 45398 48845 29292 736 46043 76410 85912 19994 58956 37013 36865 15518 96866 40781 95901 17619 64216 28401 5935 68251 69070 56621 3715 21888 98988 78900 55190 7562 20941 76713 76826 56695 55717 36366 79621 93982 26337 53666 86769 3605 80256 34796 24421 74663 74352 89791 53140 29170 831 62620 18303 88887 50830 44268 34435 19544 68250 84072 52441 96431 8811 53351 92264 4627 60443 71008 63682 68551 6128 56664 19795 26647 93356 91287 5111 59140 81244 97752 90735 73389 72164 58985 52798 76334 67386 11329 62659 21066 66552 27150 61732 96561 79192 67508 20641 92495 16954 1140 53531 75905 47840 67308 69520 64270 85773 50708 484 47358 44745 52950 47640 87407 74201 49145 74676 32570 10363 80040 3968 83614 41961 81434 54118 72132 12589 42590 93090 66228 21395 22198 33325 78648 34962 77448 4301 80062 22856 61579 21274 56109 4567 58816 56144 72938 80555 51977 59639 92724 30774 61517 93558 39378 12336 48511 52671 75793 68763 31602 55507 89978 13072 86574 37003 13607 21267 40874 36741 76487 16862 62959 30352 91182 57940 65499 86137 43151 28828 63861 6891 89723 33950 45743 82195 38068 44815 60627 62177 86364 1571 76190 82221 35703 41661 89474 28379 89193 72426 16370 17840 92204 65224 96870 8629 30022 43326 25944 74421 65191 53088 37574 65504 87351 5050 63305 1690 43088 71251 86191 18110 54418 94668 52839 6817 27026 90411 40636 31075 24859 69924 33371 68535 2286 34177 94568 95057 65834 66718 80675 9533 69682 70128 28161 84932 54771 2957 80541 44187 48687 80460 45468 14668 49113 56330 41727 58059 5440 63232 59303 44146 29783 88441 37671 92919 48908 42059 44254 72594 46139 21514 38270 50782 86344 31140 32471 88603 84118 86449 85026 36445 13856 26264 6908 14785 38365 94000 4 34286 78115 23617 35410 58549 14055 79160 86973 92689 54126 27114 44937 12417 12980 43239 39249 37209 94417 62905 48050 81331 50327 69974 75710 50103 84000 19335 83916 30522 7437 80570 27563 28709 14553 20120 76091 9363 20518 24034 32219 79818 73583 46679 76351 79925 6865 69567 29669 31263 97362 82745 65632 69281 29504 24675 9145 32392 3425 99415 97492 89543 65034 66201 79580 64276 57140 99405 47615 25589 47783 33205 56994 28849 76401 46269 34072 11009 73673 32948 84445 46413 32817 66150 65494 14404 52692 6540 92796 65180 43137 34832 12957 92907 3173 26840 95959 31960 60218 1155 89602 25948 12365 77010 84514 63098 11586 64033 12357 22938 48624 63837 50406 93926 63146 51615 86823 66444 80201 66679 81784 73705 49703 79385 68830 84982 92884 30635 59044 11008 10500 61238 41116 1494 41835 9997 40070 2444 62982 8011 84490 48610 17362 78182
87455 72390 86422 4399 33134 15676 3762 26172 9420 50728 80688 48597 31861 94451 98031 97617 25343 68439 59499 91171 79372 56889 30878 11272 27046 24382 32454 86249 36677 25121 89994 1948 16086 13881 25670 80454 9893 35574 15548 5357 86521 71320 55470 43604 89610 75114 89020 4649 4589 78326 93964 37159 2780 92987 92889 51365 73331 23774 17172 41268 45962 53260 40607 66193 54907 50977 76290 66351 95350 78158 63111 25476 18354 45310 60785 9004 55191 27075 13023 47483 24062 99701 56139 90736 44460 42752 61044 63149 9760 97462 47213 60970 52764 59842 17000 71610 8907 58143 74758 33597 52751 1366 35481 92802 47972 60639 78459 72146 19673 36223 7798 27635 35963 97393 16314 77690 82277 57115 31514 63165 56708 28524 11164 12903 38721 95419 57926 30998 10913 89180 56202 44111 446 47513 85114 55090 58576 40931 11134 18 48527 46732 11610 58191 4134 76743 86093 1408 67489 57431 87733 8776 74199 68173 51971 97537 20985 14752 43610 16481 43539 76802 7213 60951 89659 9938 5521 14810 84603 71639 58826 73490 43921 12764
