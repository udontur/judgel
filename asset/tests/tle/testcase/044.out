16819235314535
